templates/templates/docs/shared/_docs.html

```diff
@@ -98,6 +98,30 @@ <h3 class="p-side-navigation__heading">{{ nav_group.navlink_text }}</h3>
     </aside>
 
     <main class="col-9">
+      {% if document.headings_map %}
+        <h1>{{ document.title }}</h1>
```

Comment:
Hmm. This is tricky. I'm not sure this should be inside the `if` statement.

I understand your reasoning - if there's no table of contents you don't want two of the same heading right underneath each other which would look especially broken.

But on the other hand, with this method you're now *requiring* the heading to be included within the body, *only if* the table of contents is disabled. So you're forcing people to write the documentation in two different ways depending on whether they want to disable the ToC.

I think we should put the `<h1>` above the `if`, and then we just have to make sure we're communicating with documentation authors ASAP to tell them about the change. I guess we need to do an audit of exactly which documentation sets are written this way.